scripts/build.ps1

```diff
@@ -364,7 +364,7 @@ function Create-VsixPackage
     Copy-Item -Recurse $legacyDir\* $packageDir -Force
 
     # Copy Legacy data collectors Related depedencies
-    $legacyDir = Join-Path $env:TP_PACKAGES_DIR "Microsoft.VisualStudio.QualityTools.DataCollectors\15.5.0-preview-1046721\contentFiles\any\any"
+    $legacyDir = Join-Path $env:TP_PACKAGES_DIR "Microsoft.VisualStudio.QualityTools.DataCollectors\15.5.0-preview-1050541\contentFiles\any\any"
```

Comment:
Should we keep versions of both `Microsoft.VisualStudio.QualityTools` and `Microsoft.VisualStudio.QualityTools.DataCollectors` packages in sync?

Reply:
I was not sure about that so did not change. What is the guidance here?